docs/sphinx/user_guide/feature/policies.rst

```diff
@@ -57,9 +57,8 @@ OpenMP Policies
 
 OpenMP Target Policies
 ^^^^^^^^^^^^^^^^^^^^^^^^
-* ``omp_target_parallel_for_exec<NUMTEAMS>`` - Execute a loop in parallel using an ``omp target parallel for`` pragma with given number of thread teams; e.g.,
-if a GPU device is available, this is similar to launching a CUDA kernel with 
-a thread block size of NUMTEAMS. 
+* ``omp_target_parallel_for_exec<NUMTHREADS>`` - Execute a loop in parallel using an ``omp target parallel for`` pragma with given number of threads per team; e.g.,
```

Comment:
I realize the original had this too, but the trailing "e.g." implies there should be an example that seems to be missing? Either delete it or add one.